test/jasmine/tests/plot_interact_test.js

```diff
@@ -142,6 +150,119 @@ describe('Test plot structure', function() {
             });
         });
 
+        describe('scatter drag', function() {
+
+            var mock = require('@mocks/scatter_simple.json'),
```

Comment:
I think if you use [`10.json`](https://github.com/plotly/plotly.js/blob/d8e27dd33270aa7e4bb01a612cc99c51fa3cb68a/test/image/mocks/10.json), you won't even have to change the expected values.